website/static/js/filepage/index.js

```diff
@@ -363,6 +363,64 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+          var editable = self.editor && self.editor.selected;
+          var viewable = self.mfrIframeParent.is(':visible');
+          if (editable || viewable){
```

Comment:
Right, but inside this block you have an `if (viewable)` and and `if (editable)` and that's it. So you should be able to safely remove the outer if statement and just keep the inner block. I.e.
```
var editable = self.editor && self.editor.selected;
var viewable = self.mfrIframeParent.is(':visible');
if (viewable){
    self.mfrIframeParent.toggle();
}
if (editable) {
    self.editor.selected = false;
}
self.revisions.selected = true;
```
*should* work exactly the same as the code you have, yes?